src/irr/asset/CGLILoader.cpp

```diff
@@ -0,0 +1,552 @@
+				case gl::INTERNAL_RGBA4: return getTranslatedFinalFormat();			//GL_RGBA4
+
+				case gl::INTERNAL_RG4_EXT: return getTranslatedFinalFormat();
+
+				// Luminance Alpha formats
+				case gl::INTERNAL_LA4: return getTranslatedFinalFormat();				//GL_LUMINANCE4_ALPHA4
+				case gl::INTERNAL_L8: return getTranslatedFinalFormat();				//GL_LUMINANCE8
+				case gl::INTERNAL_A8: return getTranslatedFinalFormat();				//GL_ALPHA8
+				case gl::INTERNAL_LA8: return getTranslatedFinalFormat();				//GL_LUMINANCE8_ALPHA8
+				case gl::INTERNAL_L16: return getTranslatedFinalFormat();				//GL_LUMINANCE16
+				case gl::INTERNAL_A16: return getTranslatedFinalFormat();				//GL_ALPHA16
+				case gl::INTERNAL_LA16: return getTranslatedFinalFormat();				//GL_LUMINANCE16_ALPHA16
+
+				// Depth formats
+				case gl::INTERNAL_D16: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT16
+				case gl::INTERNAL_D24: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT24
+				case gl::INTERNAL_D16S8_EXT: return getTranslatedFinalFormat();
+				case gl::INTERNAL_D24S8: return getTranslatedFinalFormat();			//GL_DEPTH24_STENCIL8
+				case gl::INTERNAL_D32: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT32
+				case gl::INTERNAL_D32F: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT32F
+				case gl::INTERNAL_D32FS8X24: return getTranslatedFinalFormat();		//GL_DEPTH32F_STENCIL8
+				case gl::INTERNAL_S8_EXT: return getTranslatedFinalFormat();			//GL_STENCIL_INDEX8
+
+				// Compressed formats
+				case gl::INTERNAL_RGB_DXT1: return getTranslatedFinalFormat();						//GL_COMPRESSED_RGB_S3TC_DXT1_EXT
+				case gl::INTERNAL_RGBA_DXT1: return getTranslatedFinalFormat();					//GL_COMPRESSED_RGBA_S3TC_DXT1_EXT
+				case gl::INTERNAL_RGBA_DXT3: return getTranslatedFinalFormat();					//GL_COMPRESSED_RGBA_S3TC_DXT3_EXT
+				case gl::INTERNAL_RGBA_DXT5: return getTranslatedFinalFormat();					//GL_COMPRESSED_RGBA_S3TC_DXT5_EXT
+				case gl::INTERNAL_R_ATI1N_UNORM: return getTranslatedFinalFormat();				//GL_COMPRESSED_RED_RGTC1
+				case gl::INTERNAL_R_ATI1N_SNORM: return getTranslatedFinalFormat();				//GL_COMPRESSED_SIGNED_RED_RGTC1
+				case gl::INTERNAL_RG_ATI2N_UNORM: return getTranslatedFinalFormat();				//GL_COMPRESSED_RG_RGTC2
+				case gl::INTERNAL_RG_ATI2N_SNORM: return getTranslatedFinalFormat();				//GL_COMPRESSED_SIGNED_RG_RGTC2
+				case gl::INTERNAL_RGB_BP_UNSIGNED_FLOAT: return getTranslatedFinalFormat();		//GL_COMPRESSED_RGB_BPTC_UNSIGNED_FLOAT
+				case gl::INTERNAL_RGB_BP_SIGNED_FLOAT: return getTranslatedFinalFormat();			//GL_COMPRESSED_RGB_BPTC_SIGNED_FLOAT
+				case gl::INTERNAL_RGB_BP_UNORM: return getTranslatedFinalFormat();					//GL_COMPRESSED_RGBA_BPTC_UNORM
+				case gl::INTERNAL_RGB_PVRTC_4BPPV1: return getTranslatedFinalFormat();				//GL_COMPRESSED_RGB_PVRTC_4BPPV1_IMG
+				case gl::INTERNAL_RGB_PVRTC_2BPPV1: return getTranslatedFinalFormat();				//GL_COMPRESSED_RGB_PVRTC_2BPPV1_IMG
```

Comment:
you can return a special error on those, Vulkan doesn't distinguish between alpha and non alpha containing PVRTC